proccess_data.py

```diff
@@ -13,4 +14,10 @@ def process_data(subject_qualisys_json_path: Path,
 
     qualisys_dict = qualisys_json_creator(subject_qualisys_json_path, qualisys_df)
 
-    skeleton_frame_marker_dimension_npy = qualisys_to_generic_skeleton_converter(qualisys_dict)
+    generic_skelly_dict = qualisys_to_generic_skeleton_converter(qualisys_dict)
+
+    select_frame = np.array([1500])
```

Comment:
what's this number? Can it be specified in `main.py`?

Reply:
this number is the "favorite frame" for debugging, it's been moved to `main.py`